index.js

```diff
@@ -58,19 +58,24 @@ app.get('/api/airports', async function(req, res) {
   var qs;
   if (searchTerm.length === 3) {
     // FAA code
-    qs = "SELECT airportname from `travel-sample` WHERE faa = '" + searchTerm.toUpperCase() + "';";
+    qs = `SELECT airportname from \`travel-sample\` WHERE faa = '${searchTerm.toUpperCase()}';`;
   } else if (searchTerm.length === 4 &&
       (searchTerm.toUpperCase() === searchTerm ||
         searchTerm.toLowerCase() === searchTerm)) {
     // ICAO code
-    qs = "SELECT airportname from `travel-sample` WHERE icao = '" + searchTerm.toUpperCase() + "';";
+    qs = `SELECT airportname from \`travel-sample\` WHERE icao = '${searchTerm.toUpperCase()}';`;
   } else {
     // Airport name
-    qs = "SELECT airportname from `travel-sample` WHERE LOWER(airportname) LIKE '%" + searchTerm.toLowerCase() + "%';";
+    qs = `SELECT airportname from \`travel-sample\` WHERE LOWER(airportname) LIKE '%${searchTerm.toLowerCase()}%';`;
   }
 
-  let result = await coll.query(qs);
-  let rows = result.rows;
+  let result, rows;
```

Comment:
For future reference, typically you would want to write this something like what I've put below.  I'm don't think it makes a huge difference since there is a lot of error handling omitted in this example as is.  Specifically, we should send the error across the wire (it also cleans up the variable scoping)

```javascript
try {
  var result = await cluster.query(qs)
  res.send({
    data: result.rows,
    context: [qs]
  })
} catch(e) {
  console.error(e)
  res.send({
    error: e,
  })
}
```